### PR TITLE
Athennian Overview

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
Athenian Overview + an edit for the spartan overview

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/special/players/athen.xml [hunk 1]
```diff
@@ -6,7 +6,9 @@
   <Identity>
     <Civ>athen</Civ>
     <GenericName>Athenians</GenericName>
-    <History>As the cradle of Western civilization and the birthplace of democracy, Athens was famed as a center for the arts, learning and philosophy. The Athenians were also powerful warriors, particularly at sea. At its peak, Athens dominated a large part of the Hellenic world for several decades.</History>
+    <History>
+        The Greek city state Athens is famous remembered a famed center for the arts, learning, and philosophy, as well as one of the earliest forms of democracy. However, they were also powerful warriors.
+        Ancient Athens rose and fell as an imperial power until 330 BC when they lost the Battle of Chaeronea to King Phillip II of Macedonia. While after that it never rose again to power, Athens still was renown for its learning and its scholars were well sought out as teachers.</History>
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/special/players/athen.xml:11]:
```suggestion
        Ancient Athens rose and fell as an imperial power until 330 BC, when they lost the Battle of Chaeronea to King Phillip II of Macedonia. While after that it never rose again to power, Athens still was renown for its learning and its scholars were well sought out as teachers.</History>
```

File: 0-ad-history-mod-package/simulation/templates/special/players/athen.xml [hunk 2]
```diff
@@ -6,7 +6,9 @@
   <Identity>
     <Civ>athen</Civ>
     <GenericName>Athenians</GenericName>
-    <History>As the cradle of Western civilization and the birthplace of democracy, Athens was famed as a center for the arts, learning and philosophy. The Athenians were also powerful warriors, particularly at sea. At its peak, Athens dominated a large part of the Hellenic world for several decades.</History>
+    <History>
+        The Greek city state Athens is famous remembered a famed center for the arts, learning, and philosophy, as well as one of the earliest forms of democracy. That wasn't to say that they weren't warriors. Athens at its height had one fo the most powerful navys in the ancient world and through conquring and colinizing built a vast empire.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/special/players/athen.xml:10]:
```suggestion
        The Greek city-state of Athens is remembered a famed center for the arts, learning, and philosophy, as well as one of the earliest forms of democracy. That wasn't to say that they weren't warriors. Athens at its height had one of the most powerful navys in the ancient world and through conquering and colinizing built a vast empire.
```

File: 0-ad-history-mod-package/simulation/templates/special/players/athen.xml [hunk 3]
```diff
@@ -6,7 +6,9 @@
...
+        Ancient Athens rose and fell as an imperial power until 330 BC, when they lost the Battle of Chaeronea to King Phillip II of Macedonia. While after that it never rose again to power, Athens still was renown for its learning and its scholars were well sought out as teachers.</History>
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/special/players/athen.xml:11]:
```suggestion
        Ancient Athens rose to an imperial power after defeating the invading Persians twice in the early fifth century BC, and fell in 330 BC, when they lost the Battle of Chaeronea to King Phillip II of Macedonia. While after that it never rose again to power, Athens still was renown for its learning and its scholars were well sought out as teachers.</History>
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Very good addition!

**Comment by TheShadowOfHassen** [comment]
Is this OK to merge?

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**